### PR TITLE
PostgreSQL settings.

### PR DESCRIPTION
**Pull request by mnagy**
This is analogous to https://github.com/openshift/mysql/pull/25
I only found two settings in v2 PostgreSQL.

### CODE REVIEW

**Comment by mnagy** [comment]
[test]

**Comment by mfojtik** [comment]
LGTM

**Comment by mnagy** [comment]
Moved configuration into it's own contrib file as per @mfojtik's offline comments. PTAL

**Comment by mnagy** [comment]
..and added forgotten documentation to readme..

**Comment by mnagy** [comment]
@mfojtik can you PTAL?

File: 9.2/run-postgresql.sh [hunk 1]
```diff
@@ -56,12 +67,8 @@ if [ "$1" = "postgres" -a ! -f "$PGDATA/postgresql.conf" ]; then
 	# PostgreSQL configuration.
```
Comment (mfojtik) [inline 9.2/run-postgresql.sh]: Pls say here that this will be **appended** to the postgresql.conf, allowing users to modify it as they want.

**Comment by mfojtik** [comment]
@mnagy one small nit, but my LGTM is still valid ;-)

**Pull request merged**